src/main/java/io/vertx/core/buffer/impl/BufferInternal.java

```diff
@@ -149,9 +155,15 @@ static BufferInternal buffer(byte[] bytes) {
   @Override
   BufferInternal setDouble(int pos, double d);
 
+  @Override
+  Buffer setDoubleLE(int pos, double d);
```

Comment:
it should return BufferInternal instead of Buffer

Reply:
I just fix that in new commit ced5ae3c034717ba5d610284c28b3e748737c32a